android/src/test/java/com/instabug/reactlibrary/RNInstabugSessionReplayModuleTest.java

```diff
@@ -136,5 +135,51 @@ public void testSetUserStepsEnabled() {
         mockSessionReplay.verifyNoMoreInteractions();
     }
 
+    @Test
+
+    public void testSetSyncCallback() throws Exception  {
+        MockedStatic mockArgument = mockStatic(Arguments.class);
+        RNInstabugSessionReplayModule SRModule = spy(new RNInstabugSessionReplayModule(mock(ReactApplicationContext.class)));
+
+        CountDownLatch latch =new CountDownLatch(1);
+        SRModule.latch=latch;
```

Comment:
I don't think we should be setting the latch on `SRModule` directly in the test, `setSyncCallback` should be doing that internally.

Also, most likely we don't need to access the `CountDownLatch` in the test so let's mark it as `private`, if we must use it in the tests for some reason then let's mark it as `@VisibleForTesting`